### ISSUE TITLE
Fatal Python error when using the CLI

### ISSUE THREAD

**Issue by amelio-vazquez-reina** [qubole/qds-sdk-py#161]
I just run into this error when running:

```
qds.py --token=$QUBOLE_KEY_AVAZQUEZ hivecmd run --script_location $(pwd)/my_query.sql 
--tags "Team=opt-team" --cluster-label "default"
```

```
...
1466703091183   -1058627497_GEN_EX      http://xxx.yyy
1466703126387   -1058627497_GEN_EX      http://xxx.yyy
1466703161399   -1058627497_GEN_EX      http://xxx.yyy
1466703197279   -1058627497_GEN_EX      http://xxx.yyy
1466703232580   -1058627497_GEN_EX      http://xxx.yyy

Fatal Python error: GC object already tracked

Current thread 0x00007fff7c065000 (most recent call first):
[1]    29397 abort      qds.py --token=$QUBOLE_KEY_AVAZQUEZ hivecmd run --script_location  --tags
```

The command ID was: `28319250` and the query completed correct as far as I can tell.

**Comment by harshshah87** [comment]
This typically means something went wrong with python GC collection when multiple threads are using this. On top of my head, I cannot say what went wrong. Will reproduce it and try to get some more information.
If you can provide any more information, like , usage of tags only results in the error or it is transient, it will help.

**Comment by harshshah87** [comment]
I am not able to repro this issue for now. If you face it again/blocked by this, please post here.
==== Couple of questions if you can answer will help me understand the use case ===
Did you see the error immediately or at a later stage where it was close to query completion time?
Did it print out lots of information on console? <something that cannot be managed>
======

**Comment by amelio-vazquez-reina** [comment]
1. I saw the error after a while (the query has finished and it was downloading data).
2. The only info on the console is the one I included above.

I'll try again today and see how it goes.
